core/domain/story_domain.py

```diff
@@ -32,12 +32,15 @@
 STORY_NODE_PROPERTY_OUTLINE = 'outline'
 STORY_NODE_PROPERTY_EXPLORATION_ID = 'exploration_id'
 
+STORY_CONTENTS_PROPERTY_STARTING_NODE_ID = 'starting_node_id'
```

Comment:
Can we call it INITIAL_NODE_ID? Consistent with init_state_name in explorations, and it feels less like an action.

Reply:
Done.